src/components/Login/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink, useNavigate } from 'react-router-dom';
+
+import { TUserRequest } from '../Registration/Registration';
+import courseApi from '../../webClient';
+
+import '../../common/css/Common.css';
+import Input from 'src/common/Input/Input';
+import { TInputField } from 'src/types/inputField';
+
+const Login = () => {
+	const navigate = useNavigate();
+	const [data, setData] = useState<TUserRequest>({
+		name: '',
+		email: '',
+		password: '',
+	});
+
+	const handleUpdate = (event) => {
+		const name = event.target.name;
+		const value = event.target.value;
+		setData((values) => ({ ...values, [name]: value }));
+	};
+
+	const loginUser = (event) => {
+		event.preventDefault();
+		courseApi
+			.post('/login', JSON.stringify(data))
+			.then(function (response) {
+				navigate('/courses');
+			})
+			.catch(function (error) {
+				// TODO add hanfler
+			});
+	};
+	const emailFieldSetup: TInputField = {
+		name: 'email',
+		label: 'Email',
+		type: 'email',
+		placeHolder: '',
+	};
+	const passwordFieldSetup: TInputField = {
+		name: 'password',
+		label: 'Password',
+		type: 'password',
+		placeHolder: '',
+	};
```

Comment:
Please collect it in an array and use below with map method and move it outside component body